### DISCUSSION TITLE
[Project Beta] bug on btn closed

### DISCUSSION THREAD

**Discussion by rodrigoKulb** [community/community#6592] [Projects and Issues] [answered]
When we click on the button to see closed projects it goes to normal projects
![Captura de tela em 2021-10-21 15-00-18](https://user-images.githubusercontent.com/5334261/138332239-e763a9c5-6fc4-40a9-b865-cf8cc2398a7c.png)
Need to add the flag `?tab=projects&q=is:closed&type=beta`

**Comment by vcaballero-salle** [comment, marked as answer]
Duplicate of #6585
